Documentation/UploadBinaryData.md

```diff
@@ -157,26 +180,27 @@ Move the content link file to the **source tree** at the location where the
 actual file is desired in the build tree. Stage the new file to your commit:
 
 ```sh
-   $ git add -- path/to/file.sha512
+   $ git add -- path/to/file.cid
 ```
 
-
-
-[Download the Content Link]: https://itk.org/ITKExamples/Documentation/Contribute/UploadBinaryData.html#download-the-content-link
 [CMake]: https://cmake.org/
-[data.kitware.com]: https://data.kitware.com/
-[girder-client]: https://girder.readthedocs.io/en/latest/python-client.html#the-command-line-interface
-[Girder web interface]: https://girder.readthedocs.io/en/latest/user-guide.html
 [Git]: https://git-scm.com/
-[hashsum_download]: https://girder.readthedocs.io/en/latest/plugins.html#hashsum-download
-[ITK collection]: https://data.kitware.com/#collection/57b5c9e58d777f126827f5a1
+[ITKData Datalad repository]: https://gin.g-node.org/InsightSoftwareConsortium/ITKData/src/main
 [ITK community]: https://discourse.itk.org/
 [ITK Sphinx Examples]: https://itk.org/ITKExamples/index.html
 [ITK Software Guide]: https://itk.org/ItkSoftwareGuide.pdf
 [solution to this problem]: https://blog.kitware.com/cmake-externaldata-using-large-files-with-distributed-version-control/
-[UploadBinaryData.sh]: ../Utilities/UploadBinaryData.sh
 [CONTRIBUTING.md]: ../CONTRIBUTING.md
 
 [Analyze format]: http://www.grahamwideman.com/gw/brain/analyze/formatdoc.htm
 [MD5 hash]: https://en.wikipedia.org/wiki/MD5
 [SHA512 hash]: https://en.wikipedia.org/wiki/SHA-2
+[Content Identifier (CID)]: https://docs.ipfs.tech/concepts/content-addressing/
+[multiformats]: https://multiformats.io/
+[IPFS]: https://ipfs.io/
+[web3.storage]: https://web3.storage/
+[ITKTestingData]: https://github.com/InsightSoftwareConsortium/ITKTestingData
+[@web3-storage/w3]: https://www.npmjs.com/package/@web3-storage/w3
+[Node.js]: https://nodejs.org/
+[Content Link Upload]: https://content-link-upload.itk.org
+[Alt Link]: https://content-link-upload.itk.eth.limo
```

Comment:
Maybe these can be put in alphabetical order so that it is easier to locate them if one needs to change a reference?

Reply:
Yes, done.